DRIVERS/VME_16Z002/vme4l-core.h

```diff
@@ -150,6 +154,7 @@ typedef void *VME4L_BRIDGE_HANDLE;
 #define DEBUG_DEFAULT 0
 #endif
 
+static int debug;
```

Comment:
Please remove it.

Reply:
But it will then not compile even with gcc 4.8, it will complain that debug is not defined.

Reply:
I think you have to add "static int debug;" to  DRIVERS/VME_16Z002/vme4l-tsi148.c and DRIVERS/VME_16Z002/vme4l-pldz002.c

Reply:
In those c files debug is defined: static int debug = DEBUG_DEFAULT;  The complain comes from vme4l-core.h header file, in line 156, there debug is used but not defined.

Reply:
I agree that debug is defined in thevme4l-tsi148.c, but not in the vme4l-pldz002.c.
Could you paste me the exact error?

Reply:
Sorry I overlooked it. Updated PR. Thanks.